Documentation/artifact-format.md

```diff
@@ -95,6 +89,38 @@ Currently there is only version 1, but this document may describe later versions
 if they are created.
 
 
+manifest
+----
+
+Format: text
+
+Contains the below content exactly:
+
+```
+Version: 2
+Date: 2017-03-13 20:15:40.800231486 +0100 CET
+SHA256:
+ 1d0b820130ae028ce8a79b7e217fe505a765ac394718e795d454941487c53d32 0000/update.ext4
+ 4d480539cdb23a4aee6330ff80673a5af92b7793eb1c57c4694532f96383b619 header.tar.gz
+ 52c76ab66947278a897c2a6df8b4d77badfa343fec7ba3b2983c2ecbbb041a35 version
```

Comment:
See this: https://tracker.mender.io/browse/MEN-1016?focusedCommentId=78528&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-78528.

Maybe we can do only checksums here in more "standard" way. I am not using any library for this, so we can decide how this should look like.